src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types';
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: Tab,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onTabSelected,
+}) => {
+  const hanldeTabSelected = (
+    tab: Tab,
+    event: React.SyntheticEvent<Element, Event>,
+  ) => {
+    event.preventDefault();
+
+    if (tab.id === selectedTab.id) {
+      onTabSelected(selectedTab);
+    }
```

Comment:
Don't call this function if id is the same 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/99403198/230886244-8b79d832-a3ad-4811-b503-a5ba9b4b866c.png">
```suggestion
    if (tab.id !== selectedTab.id) {
      return onTabSelected(tab);
    }
```